R/utils-json.R

```diff
@@ -278,10 +281,51 @@ subsetByMonth <- function(pts, onlyMonth) {
   }
   return(pts)
 }
-getYvals_approvals <- function(object, num_vals){
-  ylim <- ylim(object)$side.2[1]
-  yvals <- rep(ylim, num_vals)
-  return(yvals)
+
+ApprovalInfoY <- function(object, times, c) {
```

Comment:
you should rename "c" to something different because "c" is the name of a function and this could result in some weirdness later on